pkg/v3/observation_test.go

```diff
@@ -609,7 +609,7 @@ func TestLargeObservationSize(t *testing.T) {
 	assert.NoError(t, err, "no error in decoding valid automation observation")
 
 	assert.Equal(t, ao, decoded, "final result from encoding and decoding should match")
-	assert.Less(t, len(encoded), MaxObservationLength, "encoded observation should be less than maxObservationSize")
+	assert.Greater(t, len(encoded), MaxObservationLength, "encoded observation will exceed maxObservationSize")
```

Comment:
this test doesn't make much sense now. maybe we can remove it all together if we have a test somewhere else that verifies that observation size will not exceed MaxObservationLength in any case

Reply:
or maybe we can keep this test to verify that with max proposals, we still have non zero space left for performables which are filled later